altar/src/render/brisk.rs

```diff
@@ -149,53 +165,78 @@ impl SpriteBundle {
         }
     }
 
+    /// Given a [`SpritesheetId`], get the sprites within the current bundle using the sprite sheet id.
+    /// Returns [`None`] is there are no sprites using the current sprite sheet id.
     pub fn get_sprites_in_spritesheet(
         &self,
         ss_id: SpritesheetId,
     ) -> Option<&Vec<(Sprite, Matrix4<f32>)>> {
         self.0.get(&ss_id)
     }
 
+    /// Iterates over the bundle, each item is a tuple of [`SpritesheetId`] and a vector of all the
+    /// sprites sharing the same [`SpritesheetId`].
     pub fn iter_bundle(
         &self,
     ) -> impl Iterator<Item = (&SpritesheetId, &Vec<(Sprite, Matrix4<f32>)>)> {
         self.0.iter()
     }
 
+    /// Iterates mutably over the bundle, each item is a tuple of [`SpritesheetId`] and a vector of all the
+    /// sprites sharing the same [`SpritesheetId`].
     pub fn iter_mut_bundle(
         &mut self,
     ) -> impl Iterator<Item = (&SpritesheetId, &mut Vec<(Sprite, Matrix4<f32>)>)> {
         self.0.iter_mut()
     }
 }
 
-// Unregistered sprites will be `None` for `id`
+/// Contains all of the fields making up a [`Sprite`].
 #[derive(Debug, Clone)]
 pub struct Sprite {
-    pub spritesheet_id: SpritesheetId,
+    /// Flips the sprite's UVs along the X axis.
     pub flipx: bool,
+    /// Flips the sprite's UVs along the Y axis.
     pub flipy: bool,
+    /// The sprite's opacity, 1.0 being fully opaque, and 0.0 being fully transparent.
     pub opacity: f32,
+    /// An offset that will translate the sprite in the X direction by the given value.
     pub offx: i32,
+    /// An offset that will translate the sprite in the Y direction by the given value.
     pub offy: i32,
+    /// An offset that will translate the sprite in the Z direction by the given value.
+    pub offz: i32,
+    /// Scales the sprite by the given value.
     pub scale: f32,
+    /// Selects which frame to use within the spritesheet specified by `spritesheet_id`, indexed top
```

Comment:
is this always the case? will this always be the case once we get to loading Aseprite sprites?

Reply:
It will not, this is just my current solution. In the future this'll have to be changed, and I'll be adding a TODO, and will be rewriting the docs/changing how UVs are indexed once I add aseprite support

Reply:
Cool, might wanna add that in the docs here.